lychee-lib/src/extract/mod.rs

```diff
@@ -17,7 +17,18 @@ use plaintext::extract_plaintext;
 pub(crate) fn is_verbatim_elem(name: &str) -> bool {
     matches!(
         name,
-        "code" | "listing" | "plaintext" | "samp" | "script" | "textarea" | "xmp" | "pre"
+        "address"
```

Comment:
per https://html.spec.whatwg.org/#the-address-element

`address` elements is not really verbatim as it may contain other tags. in particular, it may contain hyperlinks such as in the example:

```html
<footer>
 <address>
  For more details, contact
  <a href="mailto:js@example.com">John Smith</a>.
 </address>
 <p><small>© copyright 2038 Example Corp.</small></p>
</footer>
```

Reply:
Good point. Will remove, thanks!

Reply:
https://github.com/lycheeverse/lychee/pull/901